### DISCUSSION TITLE
disk I/O error  code 778

### DISCUSSION THREAD

**Discussion by WanderMax** [gkd-kit/gkd#779] [Announcements]
RT
使用中出现，不清楚原因。
如果完全清除数据又好了，一段时间后又出现。

**Comment by WanderMax** [comment]
![Screenshot_20241117-180944263~01](https://github.com/user-attachments/assets/cd91bd8f-c698-457d-9c95-d35865020a40)

**Comment by lisonge** [comment]
请不要使用第三方 GitHub 客户端创建不符合模板的 issue

**Comment by lisonge** [comment]
应该是 [Android room](https://developer.android.com/jetpack/androidx/releases/room?hl=zh-cn) 框架在特定设备下的崩溃异常

我不知道如何解决，如果有人能解决，欢迎 pr
